xa/java/department-spring-jpa/src/main/java/com/oracle/mtm/sample/XADataSourceConfig.java

```diff
@@ -113,38 +110,15 @@ public DataSource getXADataSource() {
         return pds;
     }
 
-    public DataSource getDataSource() {
```

Comment:
We need to clean-up similar to this in `department-spring-jpa-multiplerm` and `department-spring-jpa-multiplerm-postgres` as well

Reply:
Bharath , we have discussed to only introduce these changes into 4 application of different types . 
Later we will introduce them to other application